modules/kargoBidAdapter.js

```diff
@@ -172,28 +172,6 @@ export const spec = {
     return spec._getCrbFromCookie();
```

Comment:
you should not be writing cookies in the first party domain. I know it is out of scope of this PR, but this type of behavior will be imminently disallowed.